verifiable-db/src/revelation/revelation_without_results_tree.rs

```diff
@@ -221,6 +167,97 @@ where
 
         let flat_computational_hash = flatten_poseidon_hash_target(b, computational_hash);
 
+        // additional constraints on boundary rows to ensure completeness of proven rows
```

Comment:
Unclear why these new assertions are needed now and weren't present before. Can you clarify ?

Reply:
These constraints are necessary to enforce completeness in batching query circuits, as detailed in the [revelation circuit description](https://www.notion.so/lagrangelabs/Batching-Query-10628d1c65a880b1b151d4ac017fa445?pvs=4#11128d1c65a880e7858ecfc9dad926ed). They weren't necessary in the old circuits, and since we replace also the revelation circuit for the old query circuits with the one for batching, that's why these constraints have been added to this circuit